SU-SOE/stanford_landing_page
# SOE-1964: Initial commit of header_370x170 LPI FC header_370x170

# READY FOR REVIEW

# Summary
- Landing page new view mode for SoE

# Needed By (Date)

# Urgency

# Steps to Test
See ticket:
https://stanfordits.atlassian.net/browse/SOE-1964?focusedCommentId=66611&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-66611

1. Create a new _Stanford Landing Page_
2. In the dropdown in the form you should see "Postcard linked"
3. Select "Postcard Linked"
4. Create one or more landing page items with all fields populated
5. Save
6. Verify that the HTML on the landing page items are as expected.

# Affected Projects or Products
- SoE

# Associated Issues and/or People
- PRs
https://github.com/SU-SWS/stanford_image_styles/pull/38
https://github.com/SU-SWS/stanford_image/pull/46
- See Code Pen: 
https://codepen.io/josephgknox/pen/zdOKVo
- JIRA ticket
- https://github.com/SU-SWS/stanford_image/pull/46
- SU-SWS/stanford_image_styles#38 
- Anyone who should be notified? @boznik 
# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
